ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -82,25 +82,23 @@ private val PaywallData.Companion.defaultTemplateBaseURL: URL
 private fun PaywallData.Companion.defaultColors(
     currentColorScheme: ColorScheme,
 ): PaywallData.Configuration.ColorInformation {
+    val colors = getThemeColors(currentColorScheme)
     return PaywallData.Configuration.ColorInformation(
-        light = getThemeColors(background = PaywallColor(colorInt = Color.White.toArgb()), currentColorScheme),
```

Comment:
I removed these hardcoded colors because we could get it with `currentColorScheme.background`